### PR TITLE
test: trying to fix deployment integration tests

### PR DESCRIPTION
**Pull request by alaeddine-13**
test: trying to fix deployment integration tests

### CODE REVIEW

File: jina/jaml/parsers/flow/v1.py [hunk 1]
```diff
@@ -75,6 +75,7 @@ def parse(
                     deployments,
                     extra_search_paths=data.get('with', {}).get('extra_search_paths'),
                     include_gateway=False,
+                    noblock_on_start=True,
```
Comment (JoanFM) [inline jina/jaml/parsers/flow/v1.py:78]: I do not understand, why would this fix the issue, if at the end you are passing `True`

Comment (JoanFM) [inline jina/jaml/parsers/flow/v1.py:78]: I do not understand, why would this fix the issue, if at the end you are passing `True`

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: jina/jaml/parsers/flow/v1.py (hunk 1 above)
Comment (alaeddine-13) [reply jina/jaml/parsers/flow/v1.py:78]: The issue is not with Fload().add('deployment.yml'). It happens when the user attempts to do `Deployment.load_config('deployment.yml')` and in this case, noblock_on_start will be set to false

**Review by JoanFM — APPROVED** [review]

File: tests/integration/deployments/test_deployment.py [hunk 2]
```diff
@@ -343,7 +343,8 @@ def _create_regular_deployment(
         args.uses_after = executor if executor else 'NameChangeExecutor'
     if uses_before:
         args.uses_before = executor if executor else 'NameChangeExecutor'
-    return Deployment(args, include_gateway=False)
+    # is this needed ?
+    return Deployment(args, include_gateway=False, noblock_on_start=False)
```
Comment (alaeddine-13) [inline tests/integration/deployments/test_deployment.py:347]:
```suggestion
    return Deployment(args, include_gateway=False)
```

**Pull request merged**